### ISSUE TITLE
Hyper error: invalid certificate: UnknownIssuer

### ISSUE THREAD

**Issue by povilasv** [databricks/click#109]
```
[dev-telecom] [none] [none] > pods
No update command specified, can't update
Couldn't get an authentication token. You can try exiting Click and running a kubectl command against the cluster to refresh it. Also please report this error on the Click github page.
Hyper error: invalid certificate: UnknownIssuer
```

```
kubectl get pods --context dev-telecom
NAME                                                              READY   STATUS             RESTARTS   AGE
alertmanager-55448f5d7-85s2s                                      2/2     Running            0          2d1h
...
```

Kube config

```

- cluster:
    certificate-authority: PATH_TO_CA.PEM
    server: https://SERVER
  name: dev
- context:
    cluster: dev
    namespace: telecom
    user: USER
  name: dev-telecom

```

**Comment by Southclaws** [comment]
Running into the exact same issue.

**Comment by alex-left** [comment]
Also I have the same error but in different circumstances
I am behind a https proxy who intercept https traffic. I have installed in the system (mac os) the CA certificate and also defined common envvars pointing to this file like CURL_CA_BUNDLE or SSL_CERT_DIR. 

But the program reject the certificate. It should be able to use the system ca certificates AND/OR be able to use a custom ca AND/OR able to disable the tls verification, regardless of kubeconfig. That is the most usual, well. 

Thanks

**Comment by nicklan** [comment]
Should be fixed by 0.6.0. Please open a new issue if you're still having trouble.
